ufl/domain.py

```diff
@@ -104,6 +108,9 @@ def ufl_coordinate_element(self):
     def ufl_cell(self):
         return self._ufl_coordinate_element.cell()
 
+    def ufl_strip_data(self):
+        return Mesh(self._ufl_coordinate_element, self._ufl_id, self._ufl_cargo)
```

Comment:
I think canonically the cargo is heavy.

Reply:
Do we know what type `cargo` is? I don't know whether I should just pass it as `None` here or to implement a `ufl_strip_data` for it.

Reply:
The docstring for it says:
> `"Return carried object that will not be used by UFL."`

so I will leave it out.